tutorials/platform/html5_shell_classref.rst

```diff
@@ -0,0 +1,266 @@
+.. _doc_html5_shell_classref:
+
+HTML5 Shell Class reference
+===========================
+
+Projects exported for Web expose the ``Engine`` class to JavaScript environment, that allows 
+fine control over engine start-up process.
+
+This API is build in an asynchronous manner and requires basic understanding 
+of `Promises <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises>`__.
+
+Engine
+------
+
+The Engine class provides methods for loading and starting exported projects on Web. For default export
+settings, this is already part of the exported HTML page. To understand practical use of Engine class, 
+see :ref:`Custom HTML page for Web export <doc_customizing_html5_shell>`.
+
+Static Methods
+^^^^^^^^^^^^^^
+
++---------+-------------------------------------------------------------------------------------------------------------------+
+| Promise | `Engine.load <#Engine.load>`__ **(** string basePath **)**                                                        |
++---------+-------------------------------------------------------------------------------------------------------------------+
+| void    | `Engine.unload <#Engine.unload>`__ **(** **)**                                                                    |
++---------+-------------------------------------------------------------------------------------------------------------------+
+| boolean | `Engine.isWebGLAvailable <#Engine.isWebGLAvailable>`__ **(** *[ number majorVersion=1 ]* **)**                    |
++---------+-------------------------------------------------------------------------------------------------------------------+
+| void    | `Engine.setWebAssemblyFilenameExtension <#Engine.setWebAssemblyFilenameExtension>`__ **(** string extension **)** |
++---------+-------------------------------------------------------------------------------------------------------------------+
+
+Instance Properties
+^^^^^^^^^^^^^^^^^^^
+
++-------------------+----------------------------------+
+| Emscripten Module | `engine.rtenv <#engine.rtenv>`__ |
```

Comment:
```suggestion
| Emscripten module | `engine.rtenv <#engine.rtenv>`__ |
```

Reply:
Or if it's a class, it should use inline code formatting (and the table bounds should be adjusted).

Reply:
Yes, it's a name of an entity in Emscripten's API. I'll add code markup.